tests/test_hello.py

```diff
@@ -5,26 +5,24 @@
 # file, You can obtain one at http://mozilla.org/MPL/2.0/.
 
 import pytest
+
 from pages.hello import HelloPage
+from pages.issue import IssuePage
 
 
-class TestHelloPage:
```

Comment:
Just wondering why you decided to move these tests outside of a `class`. We usually put tests in a class, but I suppose it doesn't necessarily add any value.